.github/workflows/main.yaml

```
@@ -2,8 +2,6 @@
 name: main
 
 on:
```

```suggestion
on:
  workflow_call:
    secrets:
      SONAR_TOKEN: { required: true }
      HAPPO_API_KEY: { required: true }
      HAPPO_API_SECRET: { required: true }
      OPENSHIFT_SERVER: { required: true }
      OPENSHIFT_TOKEN: { required: true }
      OPENSHIFT_APP_NAMESPACE: { required: true }
      OPENSHIFT_METABASE_NAMESPACE: { required: true }
      NEXT_PUBLIC_GROWTHBOOK_API_KEY: { required: true }
      CLIENT_SECRET: { required: true }
      OPENSHIFT_SECURE_ROUTE: { required: true }
      AWS_S3_BUCKET: { required: true }
      AWS_S3_REGION: { required: true }
      AWS_S3_KEY: { required: true }
      AWS_S3_SECRET_KEY: { required: true }
      AWS_ROLE_ARN: { required: true }
      CERTBOT_EMAIL: { required: true }
      CERTBOT_SERVER: { required: true }
```

You need this to able to call the workflow from `nightly`, and there you need to explicitly pass secrets, like we're doing below